src/ApplicationLogs/Store/States/TransactionLogState.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// TransactionLogState.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo;
+using Neo.IO;
+
+namespace ApplicationLogs.Store.States
+{
+    public class TransactionLogState : ISerializable, IEquatable<TransactionLogState>
+    {
+        public Guid[] NotifyLogIds { get; private set; } = Array.Empty<Guid>();
+
+        public static TransactionLogState Create(Guid[] notifyLogIds) =>
+            new()
+            {
+                NotifyLogIds = notifyLogIds,
+            };
+
+        #region ISerializable
+
+        public virtual int Size =>
+            sizeof(uint) +
+            NotifyLogIds.Sum(s => s.ToByteArray().GetVarSize());
+
+        public virtual void Deserialize(ref MemoryReader reader)
+        {
+            // It should be safe because it filled from a transaction's notifications.
+            uint aLen = reader.ReadUInt32();
```

Comment:
`ReadVarInt` and `WriteVarInt` in all of them?

Reply:
I do it this way so; there is no extra byte for type of `Integer`.